lib/mappers/adobexd/textstyle.js

```diff
@@ -1,18 +1,17 @@
 const camelcase = require('camelcase');
 const { tokenize, getFontWeight } = require('../../utils');
 
-function mapTextStyle(artboard) {
-  const { name } = artboard;
-  const [rangedStyles] = artboard.value.meta.ux.rangedStyles;
+function mapTextStyle(document) {
+  const { content } = document.representations[0];
 
   return {
     token: tokenize('textStyle'),
-    id: camelcase(name.split('/').shift()),
-    weight: getFontWeight(rangedStyles.fontFamily),
-    size: rangedStyles.fontSize,
-    family: rangedStyles.fontFamily,
-    borderBottom: rangedStyles.underline ? true : null,
-    kerning: Number(rangedStyles.charSpacing.toFixed(0)) !== 0 ? parseFloat(rangedStyles.charSpacing.toFixed(1)) : null,
+    id: camelcase(document.name.split('/').shift()),
```

Comment:
Do Sketch/Figma id's for text styles named `Heading1/Align/Left` also get truncated to only `heading1`?

Reply:
Yes

Reply:
Always? Or if the --ignore-text-style-paths flag is passed?